grafana-plugin/src/components/Webhooks/WebhookLastEventDetails.tsx

```diff
@@ -15,13 +15,22 @@ import WebhookStatusCodeBadge from './WebhookStatusCodeBadge';
 
 interface WebhookLastEventDetailsProps {
   webhook: OutgoingWebhook;
+  sourceCodeRootClassName?: string;
 }
 
-const WebhookLastEventDetails: FC<WebhookLastEventDetailsProps> = ({ webhook }) => {
+const WebhookLastEventDetails: FC<WebhookLastEventDetailsProps> = ({ webhook, sourceCodeRootClassName }) => {
   const styles = useStyles2(getStyles);
   const theme = useTheme2();
   const rows = useMemo(() => getEventDetailsRows(theme, webhook), [theme, webhook]);
 
+  const sourceCodeProps = {
```

Comment:
Maybe have them called `commonSourceCodeProps` as I imagine if we were to add new fields to the form, it is not guaranteed to have the same props or?

Reply:
makes sense, will update, thanks